data/sea/52-psv.h

```diff
@@ -435,7 +435,7 @@ static ierror_loc_t psv_read_whole_buffer (psv_config_t *cfg, psv_state_t *s)
                 error = psv_write_dropped_entity_cur (s, error);
                 if (error) {
                     const char *error_pretty = ierror_loc_pretty (error, line);
-                    const char *error_msg    = strncat ("failed to drop entity:\n", error_pretty, error_msg_size);
+                    const char *error_msg    = ierror_msg_format ("failed to drop entity:\n%s\n", error_pretty);
                     cfg->error = ierror_msg_alloc (error_msg, 0, 0);
```

Comment:
you don't need this `ierror_msg_alloc` at all, `ierror_msg_format` already does the allocation.